common/static/common/js/discussion/views/discussion_topic_menu_view.js

```diff
@@ -45,7 +45,8 @@
             renderCategoryMap: function(map) {
                 var categoryTemplate = edx.HtmlUtils.template($('#new-post-menu-category-template').html()),
                     entryTemplate = edx.HtmlUtils.template($('#new-post-menu-entry-template').html()),
-                    mappedCategorySnippets = _.map(map.children, function(name) {
+                    mappedCategorySnippets = _.map(map.children, function(nameType) {
```

Comment:
Note: I don't love the data-structure of a meaningful 2x array for JavaScript, but maybe @andy-armstrong or @bjacobel could weigh in.  I guess the alternatives are:
1. Separate child_name and child_type arrays of equal length, which isn't much better, or
2. (my preference):  a child array of objects/dicts (rather than 2x arrays), with {name: "...", type: "..."}
I think #2 would look cleaner in JavaScript, but I know it would also affect a lot of js tests.

This is food for thought, unless and until the front-end devs weigh-in.